static/js/addReviewClient.js

```diff
@@ -1,35 +1,89 @@
 const addReviewButton = document.querySelector(".add-review-button");
-let addReviewContent = document.querySelector(".add-review-content");
+const addReviewContent = document.querySelector(".add-review-content");
+const loggedOutEl = document.querySelector(".add-review-logged-out");
+const loggedInEl = document.querySelector(".add-review-logged-in");
 
 addReviewButton.addEventListener("click", () => {
   addReviewButton.classList.toggle("clicked");
   addReviewContent.classList.toggle("show");
 });
 
+let loggedIn = false;
+let sendReviewToken;
+
+document.querySelector("#login").addEventListener("click", async () => {
+
+  let username = document.querySelector("#username").value;
+  let password = document.querySelector("#password").value;
+  const bufferedInput = btoa(username + ":" + password);
+  try {
+    const response = await fetch(
+    new Request("/api/token", {
+      method: "POST",
+      headers: {
+        Accept: "application/json",
+        "Content-Type": "application/json",
+        "Authentication": 
+        "Basic " + bufferedInput,
+      },
+    }));
+    if (response.status === 401) {
+      alert("Fel lösenord eller användarnamn");
```

Comment:
Would probably suggest some other type of alert if the project was more "serious" and focused around those things.

Reply:
For sure. I just remembered I hadn't added an indication for wrong password so this was my two second solution. 😁